### DISCUSSION TITLE
Alternate Image Formats?

### DISCUSSION THREAD

**Discussion by TCArknight** [Obsidian-TTRPG-Community/fantasy-statblocks#324] [Feature Requests]
While the image in the small circle works well for most of the statblocks, would it be possible to add a couple of additional stylings? One in particular I'm thinking of would be rectangular (portrait) and the height of whatever group it is associated with (or the full height if not in a group).

**Comment by Ascariel1984** [comment]
I think you can easily do this with a css snippet! I styled my image like this:

`.statblock .image-container .image img{border-radius: 10%; object-position: top;}`
`.statblock .image-container .image {  
    border: 0px solid;
    border-radius: 0%;
    width: 300px;
    height: 300px;
}`

If you want the height to match the group, you need to do something like height: 100% and maybe adjust padding/margin. Sorry I am not an CSS expert, just learned it myself.

**Reply by TCArknight** [reply]
Thank you! That worked perfectly.

Is there an easy way to have it change depending on which template I may be using? Say the original round image for a note called "NPC Summary" and the change above for the note "PC Summary" ?

**Reply by Ascariel1984** [reply]
Are you using a different layout or monster type for NPC Summary and PC Summary? Then this can be distinguished with CSS. See the documentation for this.
